src/stacks-hierarchy/Root_TicketAssistantStack/Root_TicketAssistantStack.tsx

```diff
@@ -29,14 +30,24 @@ type Props = RootStackScreenProps<'Root_TicketAssistantStack'>;
 export const Root_TicketAssistantStack = ({navigation}: Props) => {
   const [activeTab, setActiveTab] = useState(0);
   const {bottom: safeAreaBottom} = useSafeAreaInsets();
+  const analytics = useAnalytics();
   const {theme} = useTheme();
   const [previousTab, setPreviousTab] = useState<any>();
   return (
     <TicketAssistantContextProvider>
       <FullScreenHeader
         leftButton={
           activeTab === 0
-            ? {type: 'close'}
+            ? {
+                type: 'close',
+                onPress: () => {
+                  navigation.pop();
+                  analytics.logEvent(
+                    'Ticket assistant',
+                    'Closed from welcome screen',
+                  );
+                },
+              }
```

Comment:
It is not optimal that we need to specify the navigation event when we wan't to just specify the log event. It is ment that we just say `type: 'close'`, and the navigation event is handled in the screen header.

I would rather that you may send an logging context in the header button props, something like this: 
```ts
{
  type: 'close',
  logEventContext: 'Ticket assistant'
}
```

And in the header button an event can be logged when clicked. Something like this:
```ts
if (logEventContext) {
  analytics.logEvent(
    logEventContext
    'Header button of type ${type} pressed',
  );
}
```